src/main/java/seedu/address/logic/parser/SearchParser.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.logic.parser;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.IncorrectCommand;
+import seedu.address.logic.commands.SearchCommand;
+
+public class SearchParser {
+	private static final Pattern CMD_PATTERN =
+            Pattern.compile("\\s*\"(?<keyword>[^\"])"+"$");
+
+
+	public Command parse(String args){
+
+		final Matcher matcher = CMD_PATTERN.matcher(args);
+    	if (!matcher.matches()) {
+            return new IncorrectCommand(String.format(MESSAGE_INVALID_COMMAND_FORMAT, SearchCommand.MESSAGE_USAGE));
+        }
+    	final String keyword = matcher.group("keyword").trim();
```

Comment:
You can take a look at the code for [prepareFind](https://github.com/CS2103AUG2016-T11-C4/main/blob/279715e24dcb8714cd4cf5355d1afde61087ac17/src/main/java/seedu/address/logic/parser/Parser.java#L236) for reference.